linkwarner/linkwarner.py

```diff
@@ -478,6 +478,8 @@ async def on_message(self, message: discord.Message, *, edit: bool = False) -> N
                     channel.id,
                     guild.id,
                 )
+            except discord.NotFound:
+                pass  # Would've been deleted previously
```

Comment:
"Would've been deleted previously" suggests that *we* (the cog) removed the message earlier, but as far as I can tell that cannot happen as we return at the end of the loop's code.
Because of that, this comment would make more sense to me when reading this code in the future:
```suggestion
                # message had been removed before we got to it
                pass
```